src/clj/game/cards/events.clj

```diff
@@ -2273,8 +2273,8 @@
 (defcard "Push Your Luck"
   (letfn [(corp-choice [spent]
             {:player :corp
-             :waiting-prompt "Corp to guess even or odd"
-             :prompt "How many credits were spent?"
+             :waiting-prompt "Corp to choose an option"
+             :prompt "Even or odd?"
```

Comment:
Changed the text of this prompt because the answer is binary and not related to the amount of credits spent.